### PR TITLE
Tlportal 125

### PR DESCRIPTION
**Pull request by gsilver**

### CODE REVIEW

**Comment by gsilver** [comment]
@dlhaines - let me remove the extra factory that crept in

**Comment by gsilver** [comment]
@dlhaines - factory and references removed - ready to merge

**Comment by dlhaines** [comment]
Checking what happens if I reply to a PR email.

File: UI/index.erb [hunk 1]
```diff
@@ -5,6 +5,10 @@
         <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
```
Comment (dlhaines) [inline UI/index.erb]: Should this be referencing a local asset rather than a CDN?

**Comment by gsilver** [comment]
@dlhaines \
rebased and fixed conflicts

**Pull request merged**